oozie/src/main/java/org/apache/falcon/oozie/OozieBundleBuilder.java

```diff
@@ -106,11 +108,24 @@ public Properties build(Cluster cluster, Path buildPath) throws FalconException
         properties.setProperty(AbstractWorkflowEngine.NAME_NODE, ClusterHelper.getStorageUrl(cluster));
 
         //Add libpath
-        String libPath = getLibPath(buildPath);
-        if (libPath != null) {
-            properties.put(OozieClient.LIBPATH, getStoragePath(libPath));
+        String path = getLibPath(buildPath);
+        if (StringUtils.isNotBlank(path)) {
+            String storageLibPaths = null;
```

Comment:
Should StringBuilder be used instead of String to build the String for storageLibPaths.